source/includes/administration/_service_connections.md

```diff
@@ -59,3 +59,92 @@ Attributes | &nbsp;
 `type`<br/>*string* | The type of the service connection.
 `status`<br/>*Object* | Status of the service connection. Tells you if the service is up.<br/>*includes*: `lastUpdated`, `reachable`
+
+<!-------------------- GET APIINFO -------------------->
+
+### Retrieve API credentials
+
+`GET /{serviceConnectionId}/apiInfo`
+
+```shell
+# Retrieve your API credentials
+curl -X GET \
+  "https://system.cloudmc-dev.cloudops-devteam.com/rest/services/connections/5b0172cf-00bd-43de-b2e8-9aaf217c8c35/apiInfo?environmentId=ce3b76e6-c4e1-4e11-850d-e31a3055dc8e" \
+  -H "MC-Api-Key: your_api_key" \
+# Response body example
+```
+```json
+{
+    "data": {
+        "canRegenerateCredentials": true,
+        "fields": [{
+            "key": "endpoint",
+            "value": "https://your_endpoint.com/auth",
+            "sensitive": false
+        }, {
+            "key": "api_key",
+            "value": "my_api_key",
+            "sensitive": true
+        }, {
+            "key": "secret_key",
+            "value": "my_secret_key",
+            "sensitive": true
+        }, {
+            "key": "project_id",
+            "value": "079bdead-61b5-4b38-86ed-dbbf963808ec",
+            "sensitive": false
+        }]
+    }
+}
+```
+
+Retrieve your API keys for some environment within a service. A user can only access their own API keys, and only for environments where they are a member.
+
+Query Parameters (*required*) | &nbsp;
+---------- | -----
+`environmentId`<br/>*UUID* | The id of the environment.
+
+
+Attributes | &nbsp;
+---- | -----------
+`canRegenerateCredentials`<br/>*boolean* | True if the user can regenerate their API keys for the service.
+`fields`<br/>*Array[object]* | An array of key-value pairs describing the users' API keys for the service.<br/>*includes*: `key`, `value` and `sensitive`
+
+<!-------------------- GET PARAMETERS -------------------->
+
+### Retrieve connection parameters
+
+`GET /{serviceConnectionId}/parameters`
+
+```shell
+# Retrieve the connection parameters
+curl -X GET \
+  "https://system.cloudmc-dev.cloudops-devteam.com/rest/services/connections/5b0172cf-00bd-43de-b2e8-9aaf217c8c35/parameters" \
+  -H "MC-Api-key: your_api_key" \
+# Response body example
+```
+```json
+{
+    "data": [{
+        "parameter": "jsonCredentials",
+        "id": "ee7f5ba3-3efc-4efd-819b-1dd947b3473a",
+        "value": "JSON representation of credentials here",
+        "serviceConnection": {
+            "id": "d461e683-30c9-4b4e-bab6-def1a3575227"
+        }
+    }, {
+        "parameter": "billingAccountId",
+        "id": "cd404ae3-884d-4b38-a9b7-cd1bba5c0a46",
+        "value": "02C9H96Q-2H1JS2-K9SJD8",
+        "serviceConnection": {
+            "id": "d461e683-30c9-4b4e-bab6-def1a3575227"
+        } 
+    }]
+}
+```
+
+Retrieve a service connection's parameters, used to create and manage connections to services. To retrieve the parameters, you need to be an operator or be granted permissions to manage service connections.
```

Comment:
**(Line 2):** You can retrieve these parameters only if you are assigned a role that has the `Manage Service connections` permission granted to it.